client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -13,15 +14,34 @@ import { getFeaturedImageId } from 'lib/posts/utils';
 import Accordion from 'components/accordion';
 import EditorDrawerWell from 'post-editor/editor-drawer-well';
 import FeaturedImage from 'post-editor/editor-featured-image';
+import FeaturedImageDropZone from 'post-editor/editor-featured-image/dropzone';
+import isDropZoneVisible from 'state/selectors/is-drop-zone-visible';
 
 class EditorDrawerFeaturedImage extends Component {
+	static propTypes = {
+		site: PropTypes.object,
+		post: PropTypes.object,
+		translate: PropTypes.func,
+		isDrawerHidden: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		isDrawerHidden: false,
+	};
+
+	state = {};
+
 	constructor( props ) {
 		super( props );
-		this.startSelecting = () => this.setState( { isSelecting: true } );
-		this.endSelecting = () => this.setState( { isSelecting: false } );
-		this.state = { isSelecting: false };
+
+		this.state = {
+			isSelecting: false
+		};
```

Comment:
`state` is already being init'd just above (`state = {}`), so it seems right to move `isSelecting: false` into that, and ditch `constructor` altogether.